### DISCUSSION TITLE
Fastboot procedure: "DO NOT PROCEED" Ok, and then what?

### DISCUSSION THREAD

**Discussion by Mofef** [Hypfer/Valetudo#2140] [Q&A / Support] [answered]
### Are you already using Valetudo?

No, but in the process of starting it

### Your questions

Hi,

I'm trying to mentally prepare myself for the fastboot install option on my Dreame X40. The only part that I'm still really uneasy about is the repeated
> Fastboot should confirm this action with `OKAY`. If it doesn’t, **DO NOT PROCEED**.

So let's say there is no `OKAY`... There won't be enough time to ask stupid questions then, because the watchdog timer will be ticking. 
So I guess this time is now: What should one do instead of proceeding? Repeat the last command? Start over? Reboot? Pray until the timer demands it's sacrifice and ask for unbricking help afterwards?

It would also help to have an estimate of how long the combined runtime of all comands is usually. In other words: How many of these daunting 160s are left  for thinking and typing?

Thank you so much!

**Comment by vivia** [comment, marked as answer]
First of all, you should copy the commands to run in the right order into a text file and paste from there. The "thinking and typing" is a big unknown that can eat a lot of unpredictable time depending on your mood, so let's plan for it.

There is also the Telegram Valetudo group where you can seek help. You can already join now, before something goes wrong, in order to be prepared and not waste time. If something goes wrong, just hold down the power button until the robot turns off, to prevent it from rebooting continuously, and ask in the Telegram group. The answer will depend on what your specific error message will be.

**Reply by Mofef** [reply]
Thanks for all the advice! In the meanwhile I'm a happy Valetudo owner.
I joined the Telegram group and prepared the commands in a text file. Additionally I had a stopwatch running to give me a clear and easy indication of the time passed. 
Luckily everything went smooth so I can't add anything for the unlucky case.

Just in case anyone else is wondering how much runtime the commands themselves take: On my X40 Ultra it was definitely less than 20s. probably barely 10s. Your milage may vary!
